packages/generator/src/action-function-import/import.ts

```diff
@@ -127,7 +131,11 @@ export function importDeclarationsFunction(
     service.oDataVersion
   );
 }
+/* eslint-disable valid-jsdoc */
```

Comment:
only needed once per file if you use `eslint-disable` instead of `eslint-disable-next-line`